### ISSUE TITLE
[Question] Run through composer script

### ISSUE THREAD

**Issue by robsontenorio** [pestphp/pest#14]
Once we don't have Discussions tab:

I use `pcov` to run coverage reports. So i just enable it when need, to avoid performance issues when TDDing.

```bash
 "scripts": {
        "pest": "./vendor/bin/pest",
        "coverage": "php -dpcov.enabled=1 ./vendor/bin/pest --coverage --colors=always",
}
```

RUN `composer coverage`

Colors works as expected on test output

![image](https://user-images.githubusercontent.com/118955/82624145-3d1fbf80-9bb8-11ea-9fd3-e4fafb06cbc5.png)

But coverage section have no colors.

![image](https://user-images.githubusercontent.com/118955/82624092-29745900-9bb8-11ea-9650-8dccc4227a02.png)

Is there any workarounds?

**Comment by michaeldyrynda** [comment]
Try with the `--ansi` flag

**Comment by robsontenorio** [comment]
![image](https://user-images.githubusercontent.com/118955/82625241-08613780-9bbb-11ea-8eec-79913a1df7f0.png)

![image](https://user-images.githubusercontent.com/118955/82625258-0f884580-9bbb-11ea-9b6a-10e012d5e5c3.png)

**Comment by robsontenorio** [comment]
Also phpunit does not accpet `--ansi` flag

![image](https://user-images.githubusercontent.com/118955/82625460-8c1b2400-9bbb-11ea-8bd5-04ee88cc5e18.png)

**Comment by nunomaduro** [comment]
Fixed on `v0.1.5`.
